config/labels-annotations.js

```diff
@@ -22,7 +22,10 @@ export const RIO = { STACK: 'rio.cattle.io/stack' };
 
 export const CERTMANAGER = { ISSUER: 'cert-manager.io/issuer-name' };
 
-export const STORAGE = { DEFAULT_STORAGE_CLASS: 'storageclass.kubernetes.io/is-default-class' };
+export const STORAGE = {
+  DEFAULT_STORAGE_CLASS:      'storageclass.kubernetes.io/is-default-class',
+  BETA_DEFAULT_STORAGE_CLASS: 'storageclass.beta.kubernetes.io/is-default-class'
```

Comment:
From Vince's comment it looks like `BETA_DEFAULT_STORAGE_CLASS` can be removed

Reply:
I'm leaving this in for now because of https://github.com/rancher/dashboard/pull/2412#issuecomment-785209533